functions.php

```diff
@@ -11,7 +11,7 @@
  */
 define( 'VENDD_NAME', 'Vendd' );
 define( 'VENDD_AUTHOR', 'Sandhills Development, LLC' );
```

Comment:
Can we go ahead and change this to Easy Digital Downloads?

Reply:
Done.